src/mbgl/geometry/pbf_geometry.hpp

```diff
@@ -8,10 +8,10 @@
 
 namespace mbgl {
 
-class Geometry : private util::noncopyable {
+class PBFGeometry : private util::noncopyable {
 
 public:
-    inline explicit Geometry(pbf& data);
+    inline explicit PBFGeometry(pbf data);
```

Comment:
I intentionally changed from a reference to a value, as `pbf` is intended to be passed by value and at the call site was being manually copied to a temporary. In any case, I'm about to push a commit that inlines `PBFGeometry` entirely.